### ISSUE TITLE
Overlay disappears

### ISSUE THREAD

**Issue by Ronan-V** [maheshmnj/searchfield#83]
First I'd like to thank you for your beautiful work on this package, you are a great help for many people like me.

Here is the bug i'm facing.
Following 0.8.1, the suggestions overlay appears quickly before disappearing when the keyboard is shown and resizeToAvoidBottomSheet is false.

Here is a simple page code to reproduce it:

```
class MainPage extends StatelessWidget {
  const MainPage({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
        resizeToAvoidBottomInset: false,
        body: SearchField(
            suggestions: ['ABC', 'DE', 'DFHK']
                .map<SearchFieldListItem>(
                    (e) => SearchFieldListItem(e, child: Text(e)))
                .toList()));
  }
}

**Comment by maheshmnj** [comment]
Hello @Ronan-V Thanks for reporting the bug will try to fix it asap.

**Comment by maheshmnj** [comment]
@Ronan-V This has been fixed in v0.8.3.
Thank you for using this package and helping improve it with your report.

**Comment by savdev-ua** [comment]
> @Ronan-V This has been fixed in [v0.8.3.](https://pub.dev/packages/searchfield/changelog) Thank you for using this package and helping improve it with your report.

Hi @Ronan-V ,
How to fix error on scroll the page with overlay? When suggestions are shown, if you scroll page to another direction than suggestions will jump in another place. In previous versions there was a boolean called 'hasOverlay : false' that solved this issue. How to fix it now?

**Comment by maheshmnj** [comment]
Hey @savdev-ua, I am not sure I understand your issue completely, would you please file a separate issue with more details and a sample code (if applicable)? Btw Overlays are enabled by default in the latest release.
